### PR TITLE
idevice*, iproxy: add pages

### PR DESCRIPTION
**Pull request by SirMonteiro**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/idevicediagnostics.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`idevicediagnostics diagnostics`
+
+- Print mobilegestalt key values:
+
+`idevicediagnostics mobilegestalt {{key1}} {{key2}}`
+
+- Shutdown, Restart or Sleep the device:
+
+`idevicediagnostics shutdown[restart|sleep]`
```
Comment (gutjuri) [inline pages/common/idevicediagnostics.md:16]:
```suggestion
- Shutdown, restart or sleep the device:

`idevicediagnostics {{shutdown|restart|sleep}}`
```

**Review by gutjuri — APPROVED** [review]
Thanks! Looks mostly good to me, but I wonder whether we should put links to the individual commands' man pages in the more information field.

**Review by gutjuri — APPROVED** [review]
Fine from my side, thanks!

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. LGTM after the above suggestion.

File: pages/common/idevicebackup.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# idevicebackup
+
+> Create or restore backups for iOS devices.
```
Comment (spageektti) [inline pages/common/idevicebackup.md:3]:
```suggestion
> Create or restore backups for iOS devices.
```

**Pull request merged**